ast/desugar/Desugar.cc

```diff
@@ -1261,6 +1267,33 @@ TreePtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what) {
                     }
                 };
 
+                UnorderedSet<core::NameRef> hashKeySymbols;
+                auto &gs = dctx.ctx.state;
+                for (auto &key : keys) {
+                    auto lit = ast::cast_tree_const<ast::Literal>(key);
+                    core::NameRef nameRef;
+                    if (!lit) {
+                        continue;
+                    }
```

Comment:
```suggestion
                    auto lit = ast::cast_tree_const<ast::Literal>(key);
                    if (lit == nullptr) {
                        continue;
                    }
                    
                    core::NameRef nameRef;
```

Reply:
Comparing directly against `nullptr` makes the code easier to read (reinforces that it's a pointer)